### PR TITLE
CEPH-83575917: Test archive site on/off on an upload of 5K objects

### PR DESCRIPTION
**Pull request by viduship**
Automate a full sync of an RGW bucket to the archive zone having at least 5000 objects.

- https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83575917
- https://issues.redhat.com/browse/RHCEPHQE-12010
- dependent ceph-qe-scripts PR https://github.com/red-hat-storage/ceph-qe-scripts/pull/513

logs:

http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-UOTHQW

### CODE REVIEW

**Comment by mkasturi18** [comment]
Adding DNM flag until https://github.com/red-hat-storage/ceph-qe-scripts/pull/513 is merged

File: suites/reef/rgw/tier-2_rgw_ms_archive_with_haproxy.yaml [hunk 1]
```diff
@@ -568,3 +568,29 @@ tests:
...
+          config:
+            cephadm: true
+            commands:
+              - "ceph orch stop rgw.shared.arc"
+            timeout: 120
+      desc: Powerering off archive zone with IOs in progress
+      module: exec.py
+      name: Powerering off archive zone with IOs in progress
+
+  - test:
```
[the remark this answers is not in the document]
Comment (viduship) [reply suites/reef/rgw/tier-2_rgw_ms_archive_with_haproxy.yaml:583]: done

File: suites/reef/rgw/tier-2_rgw_ms_archive_with_haproxy.yaml [hunk 2]
```diff
@@ -568,3 +568,29 @@ tests:
       module: sanity_rgw_multisite.py
       name: test versioning suspend on archive
       polarion-id: CEPH-83575578
+  - test:
+      abort-on-fail: true
```
[the remark this answers is not in the document]
Comment (viduship) [reply suites/reef/rgw/tier-2_rgw_ms_archive_with_haproxy.yaml]: @mkasturi18 removed the abort-on-fail

File: suites/reef/rgw/tier-2_rgw_ms_archive_with_haproxy.yaml [hunk 3]
```diff
@@ -568,3 +568,29 @@ tests:
...
+      clusters:
+        ceph-arc:
+          config:
+            cephadm: true
+            commands:
+              - "ceph orch stop rgw.shared.arc"
+            timeout: 120
+      desc: Powerering off archive zone with IOs in progress
+      module: exec.py
+      name: Powerering off archive zone with IOs in progress
```
[the remark this answers is not in the document]
Comment (viduship) [reply suites/reef/rgw/tier-2_rgw_ms_archive_with_haproxy.yaml]: @mkasturi18 I have added the same Polarian test case ID as below.

**Pull request merged**